### ISSUE TITLE
Compiles even if there is no change in any *.less file

### ISSUE THREAD

**Issue by medokin** [gpoitch/ember-cli-less#28]
Ember: 2.0.1
Ember-Cli: 1.13.8

app.less

``` css
@import "pod-styles";
@import 'bower_components/semantic/src/semantic';
```

Console output:

```
Livereload server on http://localhost:49152
Serving on http://localhost:4200/

Build successful - 18550ms.

Slowest Trees                                 | Total
----------------------------------------------+---------------------
LessCompiler                                  | 9885ms
Concat: Vendor                                | 3949ms

Slowest Trees (cumulative)                    | Total (avg)
----------------------------------------------+---------------------
LessCompiler (1)                              | 9885ms
Concat: Vendor (1)                            | 3949ms

file changed pods\index\template.hbs

Build successful - 12134ms.

Slowest Trees                                 | Total
----------------------------------------------+---------------------
LessCompiler                                  | 8297ms

Slowest Trees (cumulative)                    | Total (avg)
----------------------------------------------+---------------------
LessCompiler (1)                              | 8297ms
```

**Comment by dwickern** [comment]
Please fix this. There are many closed issues but this is NOT fixed.
```
file changed foo.js

Build successful - 13454ms.

Slowest Trees                                 | Total
----------------------------------------------+---------------------
LessCompiler                                  | 7672ms
TreeMerger (vendor)                           | 997ms
BroccoliMergeTrees                            | 930ms

Slowest Trees (cumulative)                    | Total (avg)
----------------------------------------------+---------------------
LessCompiler (1)                              | 7672ms
TreeMerger (vendor) (1)                       | 997ms
BroccoliMergeTrees (7)                        | 994ms (142 ms)
```

**Comment by knownasilya** [comment]
Once that PR ^ is merged, this issue should be solved.

**Comment by mshindal** [comment]
This is still happening.

**Comment by mshindal** [comment]
Sorry, let me explain:

I add Bootstrap and Bootswatch to the LESS include path in `ember-cli-build.js`:
```
lessOptions: {
      paths: ['bower_components/bootswatch/paper/',
        'bower_components/bootstrap/less/']
}
```

Then in my `app.less` I import them:
```
@import 'bootstrap.less';
@import 'variables.less';
@import 'bootswatch.less';

// custom styles below...
```

This produces the output I want but it builds when any file is changed, which makes `ember server` very slow to work with:

```
file changed templates\register.hbs

Build successful - 8652ms.

Slowest Trees                                 | Total
----------------------------------------------+---------------------
LessCompiler                                  | 8040ms

Slowest Trees (cumulative)                    | Total (avg)
----------------------------------------------+---------------------
LessCompiler (1)                              | 8040ms
```

Any ideas? This happens on `ember-cli-sass` too for me.

**Comment by knownasilya** [comment]
Please comment here: https://github.com/gabrielgrant/broccoli-less-single/pull/31

**Comment by aledalgrande** [comment]
Still an issue, when I change template files, LESS files get recompiled. Am on 1.5.4.

**Comment by efx** [comment]
Opened a new issue upstream here: https://github.com/gabrielgrant/broccoli-less-single/issues/44
